kuma/javascript/src/header/search.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import Search from './search.jsx';
+
+test('Search snapshot', () => {
+    const search = create(<Search />);
+    expect(search.toJSON()).toMatchSnapshot();
```

Comment:
I like having a simple render test for components. It seems like "at least a snapshot test" would be a good standard going forward.